machine-learning/export/env.yaml

```diff
@@ -0,0 +1,25 @@
+name: base
+channels:
+  - conda-forge
+  - nvidia
+  - pytorch-nightly
+platforms:
+  - linux-64
```

Comment:
I hope ` linux-aarch64` as well?

Reply:
Eventually yes, but I had trouble installing it with aarch64 because the nightly doesn't support as many platforms. Since this is just for exporting the models, I think it's okay to only support x86_64 until torch 2.2 comes out.